dht_net.go

```diff
@@ -268,6 +270,14 @@ func (dht *IpfsDHT) sendMessage(ctx context.Context, p peer.ID, pmes *pb.Message
 	return nil
 }
 
+func (dht *IpfsDHT) getHost() host.Host {
```

Comment:
not needed? has the same visibility as the host property. this is also a weird file for it; would expect generic accessor in dht.go if needed.